### PR TITLE
Selected astyling of files with smaller changesets

### PR DESCRIPTION
**Pull request by kevingranade**
This is simply mechanical application of astyle as per doc/CODE_STYLE.txt (and a small tweak there).
Just let me know if this is too large, and _how_ too large it is, and I can respin a smaller version, but I want to get the ball rolling, because I realized I can write an astyle regression test to reject PRs that add a non-astyle compliant change to a file that was previously fully compliant.

### CODE REVIEW

File: src/damage.cpp [hunk 1]
```diff
@@ -162,7 +162,7 @@ void ammo_effects( const tripoint &p, const std::set<std::string> &effects )
     if( effects.count( "NAPALM" ) > 0 ) {
         g->explosion( p, 4, 0.7, 0, true );
         // More intense fire near the center
-        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
+        for( auto && pt : g->m.points_in_radius( p, 1, 0 ) ) {
```
Comment (Coolthulhu) [inline src/damage.cpp]: Looks like `astyle` sees those as `AND`. Any way to make it not do so?

**Comment by Coolthulhu** [comment]
By the way it would be nice to have a consistent `if` and function parenthesis padding or lack of thereof as the part of the astyle.

I use `if( x )`, some code uses `if(x)`, some `if (x)` and some even `if( x)`.
I believe astyle can un-pad and then re-pad it in the same pass.

File: src/damage.cpp (hunk 1 above)
Comment (kevingranade) [inline src/damage.cpp]: Looks like we just need to use astyle version 2.05 or later, I used 2.04.something.
I'll re-run with a recenbt version of astyle and re-PR.

**Comment by kevingranade** [comment]
I'll take a look at specifying ```if( stuff )``` as the style, I correct it to that all over the place anyway.
I'm going to redo with a more recent version of astyle and a smaller list of files that can hopefully be pushed through faster.  Good catches though.

**Pull request closed without merging**